charts/airflow/files/pod_template.kubernetes-helm-yaml

```diff
@@ -66,6 +67,10 @@ spec:
       ports: []
       command: []
       args: []
+      {{- if .Values.airflow.kubernetesPodTemplate.securityContext }}
```

Comment:
This is not valid, the `airflow.kubernetesPodTemplate.securityContext` value sets the [PodSecurityContext](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#podsecuritycontext-v1-core), not the [SecurityContext](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#securitycontext-v1-core).

Reply:
Please remove this change.